cleanlab/regression/rank.py

```diff
@@ -142,7 +141,7 @@ def _get_outre_score_for_each_label(
     *,
     residual_scale: float = 5,
     frac_neighbors: float = 0.5,
-    neighbor_metric: Optional[Union[str, Callable]] = "euclidean",
+    neighbor_metric: Optional[Union[str, Callable]] = None,
```

Comment:
Looks like previous PRs updating the choice of metric implementations forgot to switch to the correct behavior.

Reply:
what do you mean?  This change here is INCORRECT.

The metric in this get_outre_score() method should always be the Euclidean distance. Whether to use your "euclidean"-sklearn version or euclidean-scipy version I'll leave up to you. But this method should never be using cosine ever

Reply:
you could be relying on the fact that internally the features corresponding to this method will never exceed 3-D and thus it will automatically use Euclidean distance, but that is a pretty dangerous implicit assumption. If you want to rely on this assumption, it needs to be explicitly commented in the code/documentation, so that it's not forgotten in the future

Reply:
I've now ensured that this function picks a euclidean metric.